docs/www/docs/contribute/99-troubleshooting.md

```diff
@@ -41,6 +41,16 @@ In order to avoid this issue, you have a few options:
 4. Manually provide `--client-id`, `--client-secret`, `--app-name` and `--resource-group`.
 
 
+## SWA CLI - Domains that need Firewall access
+
+### Context
+
+For SWA CLI to operate perfectly, the following domains need to be allowed an access in your firewall:
```

Comment:
Instead of "operate perfectly", can we mention that if users are facing problem with the firewall, they can allowlist these instead?